mz_zip_rw.c

```diff
@@ -1834,6 +1837,14 @@ void mz_zip_writer_set_follow_links(void *handle, uint8_t follow_links) {
     writer->follow_links = follow_links;
 }
 
+int32_t mz_zip_writer_get_follow_links(void *handle, uint8_t *follow_links) {
+    mz_zip_writer *writer = (mz_zip_writer *)handle;
+    if (!follow_links)
```

Comment:
```suggestion
    if (!writer || !follow_links)
```

Reply:
This was already merged. I opened a new MR for this suggestion: #891